.github/workflows/validate-odo.yaml

```diff
@@ -13,7 +13,7 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-name: Validate Devfile stacks
+name: Validate odo
```

Comment:
The name and file name are misleading. 
This workflow is not only about odo, but there is also a job that checks for non-terminating images.